src/Chunk.ts

```diff
@@ -1046,16 +1066,18 @@ export default class Chunk {
 				}
 			} else if (variable instanceof GlobalVariable) {
 				hoisted = true;
+			} else if (variable instanceof SyntheticNamedExportVariable) {
+				auxLocal = variable.name;
 			}
 
-			const localName = variable.getName();
-
 			exports.push({
+				auxLocal,
```

Comment:
Maybe it is a `property`? I really find `auxLocal` unhelpful as it does not tell me what this value means. `property` could tell me that this is the property we are exporting. The fact that I need to introduce a local variable is of course another question.